src/accelerate/launchers.py

```diff
@@ -65,14 +72,14 @@ def notebook_launcher(function, args=(), num_processes=None, use_fp16=False, mix
                 num_processes = 8
 
             launcher = PrepareForLaunch(function, distributed_type="TPU")
-            print(f"Launching a training on {num_processes} TPU cores.")
+            print(f"Launching a training on {num_processes} TPU cores")
```

Comment:
Not sure why the period is removed here, but I like them at the end of sentences :-p

Reply:
That's from me messing with some things, put it back!